content/overlay.xul

```diff
@@ -33,11 +33,16 @@
 			if (contextMenu) {
 				contextMenu.addEventListener("popupshowing", function(event) {
 					var IT = ImageTweak.enabled(document.popupNode.ownerDocument);
-					if (IT && !(gContextMenu.onImage || gContextMenu.onVideo) && ImageTweak.pref.ContextMenu) {
+					if (IT && !(gContextMenu.onImage || gContextMenu.onVideo || gContextMenu.onAudio) && ImageTweak.pref.ContextMenu) {
 						event.preventDefault();
 						ImageTweak.DelayedExecute(function() {
 							ImageTweak.showContextMenuForElement(IT.Image, event.screenX, event.screenY);
 						});
+					} else if (gContextMenu.onAudio) {
+						var menuitems = document.querySelectorAll('menuitem[id^="context-imagetweak-"], menuseparator[id^="context-imagetweak-"]');
+						for (var i=0; i<menuitems.length; i++)
+							menuitems[i].hidden = true;
+					}
```

Comment:
There's an extra closing curly brace ( } ) here. 

Otherwise, this fix appears to work great. Thanks!